app/controllers/books_controller.rb

```diff
@@ -37,6 +37,12 @@ def update
     end
   end
 
+  def destroy
+    Book.find(params[:id]).destroy
```

Comment:
Before action destroy is find_and_assign_book, you got @book

Reply:
resolved